_posts/2025-10-22-Buildah-version-v1.42.0.md

```diff
@@ -0,0 +1,95 @@
+   * Run: reap stray processes by [@nalind](https://github.com/nalind) in [#6307](https://github.com/containers/buildah/pull/6307)
+   * imagebuildah.StageExecutor.Execute: commit more "no instructions" cases by [@nalind](https://github.com/nalind) in [#6314](https://github.com/containers/buildah/pull/6314)
+   * Rework how we decide what to filter out of layer diffs by [@nalind](https://github.com/nalind) in [#6332](https://github.com/containers/buildah/pull/6332)
+   * Mount image fails with imagestores by [@nalind](https://github.com/nalind) in [#6340](https://github.com/containers/buildah/pull/6340)
+   * Losen the dependency on go-connections/tlsconfig by [@mtrmac](https://github.com/mtrmac) in [#6328](https://github.com/containers/buildah/pull/6328)
+   * The global `--imagestore` flag can now be used when configuring storage by [@nalind](https://github.com/nalind) in [#6341](https://github.com/containers/buildah/pull/6341)
+   * The global `--transient-store` flag is now recognized by [@nalind](https://github.com/nalind) in [#6346](https://github.com/containers/buildah/pull/6346)
+   * FROM instructions which reference images using both a tag and a digest no longer trigger errors in `buildah build` with `--all-platforms`: the tag portion of the reference will be ignored by [@nalind](https://github.com/nalind) in [#6353](https://github.com/containers/buildah/pull/6353)
+   * The chroot isolation handler should now correctly use $PATH to find the binary to run when RUN instructions express the command to run in exec form, and which do not do so using an absolute path by [@nalind](https://github.com/nalind) in [#6378](https://github.com/containers/buildah/pull/6378)
+   * Parent directories created for mounts used by `buildah run` or by RUN instructions in `buildah build` will be world-readable again by [@nalind](https://github.com/nalind) in [#6381](https://github.com/containers/buildah/pull/6381)
+   * commit: always return the config digest as the image ID by [@nalind](https://github.com/nalind) in [#6426](https://github.com/containers/buildah/pull/6426)
+   * copier: ignore user.overlay.* xattrs by [@nalind](https://github.com/nalind) in [#6429](https://github.com/containers/buildah/pull/6429)
+   * Include the Sequoia crypto backend, and run integration tests with it by [@mtrmac](https://github.com/mtrmac) in [#6390](https://github.com/containers/buildah/pull/6390)
+   * build,add: add support for corporate proxies by [@userid0x0](https://github.com/userid0x0) in [#6274](https://github.com/containers/buildah/pull/6274)
+
+* Documentation:
+   * Adding mohanboddu as community manager to MAINTAINERS.md by [@mohanboddu](https://github.com/mohanboddu) in [#6339](https://github.com/containers/buildah/pull/6339)
+   
+* Vendored: 
+   * Vendor github.com/containers/common to v0.64.1 
```

Comment:
```suggestion
   * Vendor go.podman.io/common to v0.66.0 
```

Reply:
The vendor info came from the git release note creation process, it must have hiccuped somehow.  I'll tweak it.